gemini/tests/set4/tabs.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
```

Comment:
We can revert this. As there are no code changes in the file, this will be only commit noise.